watertap/data/techno_economic/autothermal_hydrothermal_liquefaction.yaml

```diff
@@ -0,0 +1,104 @@
+default:
+  energy_electric_flow_mass:
+    value: 8.71
+    units: kWh/tons
```

Comment:
Do we know if this represents metric or imperial tons?

Reply:
It should be metric tons based on their provided data, do I need to make any modifications?

Reply:
Then I think you need to use `metric_ton` or `tonne`. Check out the following link to see available pint units and naming: https://github.com/hgrecco/pint/blob/master/pint/default_en.txt

Reply:
That works although using `metric_ton` might be more explicit for a new user who might be reviewing the code.